examples/js/controls/OrbitControls.js

```diff
@@ -179,13 +172,12 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 			if ( scope.enableDamping === true ) {
 
-				thetaDelta *= ( 1 - scope.dampingFactor );
-				phiDelta *= ( 1 - scope.dampingFactor );
+				spherical.theta *= ( 1 - scope.dampingFactor );
+				spherical.phi *= ( 1 - scope.dampingFactor );
```

Comment:
It should be `sphericalDelta` in your notation.

But like I said previously, I still think the notation `thetaDelta` (or `deltaTheta`, which is even better, IMO) is better notation that what you have introduced here.

`sphericalDelta` is not a spherical coordinate -- it is an offset in phi and theta, and has no meaning otherwise.